R/output_format.R

```diff
@@ -120,6 +120,7 @@ merge_output_formats <- function (base, overlay)  {
 #'   \code{\link[knitr:opts_chunk]{opts_chunk}})
 #' @param knit_hooks List of hooks for R code chunks, inline R code, and output
 #'   (see \code{\link[knitr:knit_hooks]{knit_hooks}})
+#' @param opts_template List of templates for chunk level knitr options (see \code{\link[knitr:opts_template]{opts_template}})
```

Comment:
Please reflow comments here. If you are in RStudio, you can just press Ctrl+Shift+/